test/axes/axisTests.ts

```diff
@@ -655,4 +655,39 @@ describe("Axis", () => {
     axis.destroy();
     div.remove();
   });
+
+  describe("tickLabelAt", () => {
+    let div: d3.Selection<HTMLDivElement, void | {}, any, any>;
+    let axis: Plottable.Axis<number>;
+    let scale: Plottable.Scales.Linear;
+
+    before(() => {
+      // labels are spaced ~(300px / 10)
+      div = TestMethods.generateDiv(300, 100);
+      scale = new Plottable.Scales.Linear();
+      scale.domain([0, 100]);
+      axis = new Plottable.Axes.Numeric(scale, "bottom");
+      axis.renderTo(div);
+    });
+
+    after(() => {
+      axis.destroy();
+      div.remove();
+    });
+
+    it("returns label data when clicking on label text", () => {
+      assert.equal(axis.tickLabelAt({ x: 30, y: 10}), 10);
+      assert.equal(axis.tickLabelAt({ x: 60, y: 10}), 20);
+    });
+
+    it("calculates only in the direction of the axis", () => {
+      assert.equal(axis.tickLabelAt({ x: 60, y: 0}), 20);
+      assert.equal(axis.tickLabelAt({ x: 60, y: 50}), 20);
+      assert.equal(axis.tickLabelAt({ x: 60, y: 100}), 20);
+    });
+
+    it("returns undefined if point is outside of labels", () => {
+      assert.isUndefined(axis.tickLabelAt({ x: 10, y: 10 }));
```

Comment:
It might be valuable to repurpose `tickLabelAt` to allow clicks outside of labels (so the user can click anywhere in the area, not specifically on the label text, to get the corresponding data)